spec/asciidoctor-standoc/isobib_cache_spec.rb

```diff
@@ -106,119 +106,117 @@
 EOS
 
   ISO_123_UNDATED = <<~EOS
```

Comment:
This is being affected by https://github.com/metanorma/relaton-iso/issues/54 . i.e. the undated reference should be fetching the newest document, not the oldest. Can we not commit this but make the test still pass?

Reply:
I think no, let me revert all changes in all embedded xml in `*_spec.rb`

Reply:
reverted